userguide/storage.rst

```diff
@@ -648,6 +648,10 @@ Here are some examples:
 Export/Disconnect a Pool
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+:guilabel:`Export/Dissonnect` removes a pool from the %brand%
+system, making the disks in the pool available. The data in the pool
```

Comment:
It mainly disconnects a pool.  The disks are only made available if they are erased, but that should not be the default.
"Export/Disconnect is used to cleanly disconnect a pool from the system. This is used before physically disconnecting the pool so it can be imported on another system, or to optionally detach and erase the pool so the disks can be reused."